### ISSUE TITLE
New message/order/customer notification bubble not updated live

### ISSUE THREAD

**Issue by costads** [PrestaShop/PrestaShop#23468]
### Describe the bug
New message/order/customer notification bubble is not updated live, without refreshing the browser

### Expected behavior
I expected a notification bubble number on either a new message/customer/order without refreshing the browser like in 1.7.6 and previous years versions.

### Steps to Reproduce
Open frontpage of BO, and notice the notification bubbles does not update the numbers as you get new orders/messages.

-------

I have tried this on both my 1.7.6.1 installations, and I also tried some other 1.7.6 and even 1.6.1.24 demo of backoffice online.
You have to wait maybe a minute or so, then the bubbles will update.

I just tried this one:
https://www.prestademo.it/16/admin555/ - demo@demo.com / demodemo

https://www.prestademo.it/16/
Just send a message / create a user and then watch backoffice a minute or so, then you will see.

![109360476-98898700-7887-11eb-942e-e14667760575](https://user-images.githubusercontent.com/67175129/109576177-975c9200-7af3-11eb-92d0-87847b9c9b84.jpg)

### Additional information

- PrestaShop version: 1.7.7.1
- PHP version: 7.3

**Comment by hibatallahAouadni** [comment]
Hello @costads 

Could you please check this [comment](https://github.com/PrestaShop/PrestaShop/issues/23399#issuecomment-786800719) it may resolve your issue. And check this issue #9550 seems similar to yours :wink: 
Please check and feedback.

Thanks!

**Comment by costads** [comment]
> Hello @costads
> 
> Could you please check this [comment](https://github.com/PrestaShop/PrestaShop/issues/23399#issuecomment-786800719) it may resolve your issue. And check this issue #9550 seems similar to yours 😉
> Please check and feedback.
> 
> Thanks!


Thanks for the tip, but I tried that without luck.

**Comment by hibatallahAouadni** [comment]
Hello @costads 

I didn't reproduce your issue with PS1.7.7.1 and PS1.7.7.2, see the attached screen record below:

https://drive.google.com/file/d/1BxjAu-USWaz-qgf1Z24IiZcWSyejd55a/view

After a minute the notification appeared :wink: 
Please check and feedback.

Thanks!

**Comment by costads** [comment]
Yeah, that's strange. I don't suppose single/multishop could change the way the notification is working?
I don't guess it has anything to do with PHP versions etc, as I have 1.7.6.1 running on other domains on that server which is working fine. 

Also no errors anywhere, not even in Chrome debug.
Do you have any ideas?

**Comment by hibatallahAouadni** [comment]
Hello @costads 

Sorry but I have no idea why it's not working with you :confused: 

I have to close the issue cause we use GitHub issues only to discuss about bugs and new features in the PrestaShop project. And i seems your is not related to Core PS (it may related to your host), if you need help with your shop, please consider one
of our [support plans](https://www.prestashop.com/support).

Alternatively, you can also ask for help in the [community forums](https://www.prestashop.com/forums/) or in the [public Slack channel](https://github.com/PrestaShop/open-source/tree/master/slack).

Thank you

**Comment by costads** [comment]
I have just installed a fresh copy of PS 1.7.7.2 which is stock, and it is still not working.
I noticed the message is actually coming in. If I click on the notification bell, the message is there. So basically it is only the yellow count icon that is missing, until I refresh the browser.

I have tried it in Chrome on my work- and home computer, Opera, and lastly my Chromebook. All the same, no yellow icon, but the message is showed in the parenthesis in the modal popup.
![notify](https://user-images.githubusercontent.com/67175129/110265080-06862a80-7fbb-11eb-9cef-c418f4bd1a45.jpg)


Can that bubble be a server problem with 1.7.7.2?  PS 1.7.6.1 is working fine on the same server.
There are no errors anywhere on server logs, PS logs, and in Chrome debug.

![github1](https://user-images.githubusercontent.com/67175129/110220250-87162f80-7ec4-11eb-86f5-e93b04b3e769.png)
